test/e2e/common/setup-kubestellar.sh

```diff
@@ -112,6 +112,8 @@ wait-for-cmd '(kubectl -n wds1-system wait --for=condition=Ready pod/$(kubectl -
 
 echo "transport controller is running."
 
+wait-for-cmd 'kubectl --context imbs1 get ns customization-properties'
```

Comment:
Why did we decide that customization deserves to go into the shell testing? Why not just add a ginkgo test for customization?  
I think this customization should be removed from this test and moved to ginkgo